client/src/view/AcceptRide.js

```diff
@@ -0,0 +1,129 @@
+import React from "react"
+import styled from "styled-components";
+import Destination from "./icons/Destination";
+import Phone from "./icons/Phone";
+import Calendar from "./icons/Calendar";
+import GoogleRoutesForm from "./forms/GoogleRoutesForm";
+import Vertical from "./icons/Vertical";
+import Avatar from './Avatar';
+
+const WrapperTop = styled.div`
+	font-size: 20px;
+	font-weight: bold;
+	color: gray;
+	padding-top: 20px;
+	width: 70%;
+	position: absolute;
+`
+const Wrapper = styled.div`
+
+`
+const DivLeft = styled.div`
+	float: left;
+`
+
+const Map = styled.div`
+	float: right;
+	border: 1px solid black;
+`
+
+const AvailableDiv = styled.div`
+	width: 70%;
+	height: 17%;
+	position: absolute; 
+	top: 217px;
+	font-weight: bold;
+	background-color: #f2f2f2; 
+`
+
+const ReplyDiv = styled.div`
+	width: 70%;
+	height: 65px;
+	position: absolute; 
+	top: 307px;
+	padding-top: 10px;
+	background-color: #e6e6e6;
+`
+
+const Reply = styled.p`
+	font-size: 12px;
+`
+
+const Accept = styled.button`
+	height: 50px; 
+	width: 300px;
+	background-color: #00b33c;
+	color: white;
+	font-weight: bold;
+	font-size: 30px;
+`
+
+const Decline = styled.button`
+	height: 50px; 
+	width: 300px;
+	background-color: #cc0000;
+	color: white;
+	font-weight: bold;
+	font-size: 30px;
+`
+const Buttons = styled.div`
+	position: absolute;
+	top: 10px;
+	padding: 10px;
+	padding-left: 80px;
+	display: inline;
+`
+
+const Iframe = styled.iframe`
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 80%;
+  z-index: 1;
+`;
+
+//const placeIframe = `https://www.google.com/maps/embed/v1/search?key=${apiKey}&q=${origin}`;
+//const directionIframe = `https://www.google.com/maps/embed/v1/directions?key=${apiKey}&origin=${origin}&destination=${destination}`;
+//const { apiKey = window.REACT_APP_PLACES_API_KEY } = this.props;
+ 
+export default () => (
```

Comment:
You will want to write this like

```jsx
export default ({ origin, destination, apiKey = window.REACT_APP_PLACES_API_KEY, ...otherArgs }) => {
  const placeIframe = `https://www.google.com/maps/embed/v1/search?key=${apiKey}&q=${origin}`;
  const directionIframe = `https://www.google.com/maps/embed/v1/directions?key=${apiKey}&origin=${origin}&destination=${destination}`;
  return (
    <div>your jsx</div>
  )
}
```

Then in your storybook you can pass in those arguments like this:
```jsx
<AcceptRide origin="1900 empire bvld, Webster, NY" destination="55 North Ave, Webster, NY" />
```

Since we gave apiKey a default property it's optional to set from the storyboard. We didn't have to make apiKey a prop because we will probably never change it but it's just good practice to be able to override it.